src/libs/actions/Report.ts

```diff
@@ -635,14 +636,38 @@ function updateGroupChatName(reportID: string, reportName: string) {
             key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
             value: {
                 reportName,
-                errorFields: {
+                pendingFields: {
+                    reportName: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                },
+            },
+        },
+    ];
+
+    const successData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {
+                pendingFields: {
                     reportName: null,
                 },
             },
         },
     ];
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {
+                reportName: ReportUtils.getReport(reportID)?.reportName,
+                errors: {
+                    reportName: Localize.translateLocal('groupChat.invalidGroupChatName'),
+                },
+            },
+        },
+    ];
```

Comment:
I think we should keep `pendingFileds` so it can be grey out in offline if the error appears.

The `pendingFileds` will be cleared when we clear the error,

Reply:
Why it needs to be grayed out? The 50% opacity is only needed for unfinished actions, a failed request is still a finished action

Reply:
It's the same behavior when we edit the room name here.

https://github.com/Expensify/App/blob/db469d6e93e8eeae8c1251d1ebd9a94a6dfdd502/src/libs/actions/Report.ts#L2163

Reply:
That behavior is also wrong